ansible/roles/postgresql/defaults/main.yml

```diff
@@ -1,5 +1,5 @@
 pg_install: True
-postgresql_version: '9.1'
+postgresql_version: '9.4'
```

Comment:
Does this place also need a version change? https://github.com/dimagi/commcarehq-ansible/blob/pg94/ansible/roles/common_installs/defaults/main.yml#L1